references/validationFunctions/follow_up/relapse_interval.js

```diff
@@ -1,27 +1,39 @@
-/*
```

Comment:
@hknahal it looks like the copy right header was removed here - can you add that back in?

Reply:
I must have mistakenly deleted them! I put them back in. Also fixed the script validation as well.